server/src/main/java/org/elasticsearch/indices/cluster/IndicesClusterStateService.java

```diff
@@ -942,7 +951,13 @@ void processPendingDeletes(Index index, IndexSettings indexSettings, TimeValue t
              * Persistent parts of the index like the shards files, state and transaction logs are kept around in the
              * case of a disaster recovery.
              */
-            FAILURE
+            FAILURE,
+
+            /**
+             * The index have been closed. The index should be removed and all associated resources released. Persistent parts of the index
```

Comment:
s/have/has/

Reply:
The whole comment is a copy-paste of the CLOSED one... I fixed that